test/test_main.cpp

```diff
@@ -42,6 +42,11 @@ int main(int argc, char **argv) {
         print_usage();
         return 0;
       case 'e':
+#ifndef MANIFOLD_EXPORT
+        printf(
```

Comment:
maybe print to stderr?

Reply:
Fair, but I think since the tests will still run successfully it's more of a warning than an error.